### ISSUE TITLE
[Accessibility issue]Keyboard focus on active switch is not visible

### ISSUE THREAD

**Issue by Lausselloic** [Orange-OpenSource/Orange-Boosted-Bootstrap#1074]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

![image](https://user-images.githubusercontent.com/11272238/151817302-794463b7-a7c2-4f37-ab0f-13ff5e081d1f.png)
When browsing switch element with keyboard, when the switch is active the outline focus color is white, so on white background it can't be perceptible

### Reduced test cases

Use keyboard to focus the active switch

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

91.2.0 ESR
